src/lib/expansion/accordion.ts

```diff
@@ -0,0 +1,39 @@
+import {Directive, Input} from '@angular/core';
+import {
+  trigger,
+  state,
+  style,
+  transition,
+  animate,
+} from '@angular/animations';
+import {coerceBooleanProperty} from '../core/coercion/boolean-property';
+import 'rxjs/add/operator/takeUntil';
+
+export type CdkAccordionDisplayMode = 'default' | 'flat';
+
+/** Unique ID counter */
+let nextId = 0;
+
+/**
+ * Directive whose purpose is to manage the expanded state of CdkAccordionItem children.
+ */
+@Directive({
+  selector: '[cdk-accordion]',
```

Comment:
Also, we should make a separate `<md-accordion>`  that is probably just
```ts
@Directive({ selector: 'md-accordion'})
export class MdAccordion extends CdkAccordion { }
```
Just so users don't have to mix `md-` and `cdk-`